helpdesk/templates/helpdesk/navigation-sidebar.html

```diff
@@ -68,6 +68,15 @@
             <span>{% trans "New Ticket" %}</span>
           </a>
         </li>
+        {% if user.is_authenticated %}
+        <li class="nav-item{% if 'my-tickets' in request.path %} active{% endif %}">
+            <a class="nav-link" href="{% url 'helpdesk:my-tickets' %}">
+                <i class="fas fa-fw fa-tasks"></i>
+                <span>{% trans "My Tickets" %}</span>
+            </a>
+        </li>
+
+        {% endif %}
```

Comment:
You will have to add this to both parts of the if/then otherwise if someone sets `HELPDESK_NAVIGATION_ENABLED` to Tue or the app is used internally by the company to provide a helpdesk and someone is an admin, they cannot use the "My Tickets" view.

Reply:
Added above.